src/utils/notifyIssues.js

```diff
@@ -0,0 +1,98 @@
+'use strict'
+
+const fs = require('fs')
+const pMap = require('p-map')
+
+const { logError, logWarning } = require('../log')
+const { getPrNumbersFromReleaseNotes } = require('./releaseNotes')
+
+async function getLinkedIssueNumbers({
+  octokit,
+  prNumber,
+  repoOwner,
+  repoName,
+}) {
+  const data = await octokit.graphql(
+    `
+    query getLinkedIssues($repoOwner: String!, $repoName: String!, $prNumber: Int!) {
+      repository(owner: $repoOwner, name: $repoName) {
+        pullRequest(number: $prNumber) {
+          id
+          closingIssuesReferences(first: 100) {
+            nodes {
+              id
+              number
+            }
+          }
+        }
+      }
+    }
+    `,
+    {
+      repoOwner,
+      repoName,
+      prNumber,
+    }
+  )
+
+  const linkedIssues =
+    data?.repository?.pullRequest?.closingIssuesReferences?.nodes
+
+  if (!linkedIssues) {
+    return []
+  }
+
+  return linkedIssues.map(issue => issue.number)
+}
+
+async function notifyIssues(githubClient, owner, repo, release) {
+  let packageName
+  let packageVersion
+
+  try {
+    const packageJsonFile = fs.readFileSync('./package.json', 'utf8')
```

Comment:
When I used `require` the contents of the file got evaluated at build time and pasted in the contents of the `package.json` from the `optic-release-automation-action` rather than the repo using the action. So I replaced it with the above to make sure that the `package.json` is read at runtime.

Reply:
At the time it was at the top of the `release.js` file as `const packageJsonFile = require('../package.json')`.
If I tried the path as `'./package.json'` then I got module not found on build time.

Reply:
yes of course if you put it at the top it's not the same as requiring it in the function

Reply:
I have tried to replace `const packageJsonFile = fs.readFileSync('./package.json', 'utf8')` with `const packageJson = require('./package.json')` inside the function itself at the same line. Now I get `Error: Error: Cannot find module './package.json'` at runtime rather than build time. My understanding was that I should use the same path for both implementation. Any suggestions?

Reply:
I'm not against reading the file but I'm confident there's a way to require the file. try github.workspace variable https://docs.github.com/en/actions/learn-github-actions/contexts#github-context

Reply:
which means it works, I assume?

Reply:
Yes, thank you for the pointers. The only thing left now is to refactor the relevant tests.